instrumentation/jersey-server/src/main/java/brave/jersey/server/TracingApplicationEventListener.java

```diff
@@ -75,6 +78,34 @@ public RequestEventListener onRequest(RequestEvent requestEvent) {
       return request.getHeaderString(name);
     }
 
+    /**
+     * This returns the matched template as defined by a base URL and path expressions.
+     *
+     * <p>Matched templates are pairs of (resource path, method path). This code skips redundant
+     * slashes from either source caused by Path("/") or implicitly by Path("").
+     */
+    @Override public String route(ContainerResponse response) {
```

Comment:
@jkschneider fyi. feel free to steal this.. managed to get to the bottom of why there were double-slashing. jersey actually normalizes "" to "/" btw. This is a lot more efficient in benches

Reply:
added a commit.. maybe wait until merged :)